### DISCUSSION TITLE
How to cut top and bottom before trimming?

### DISCUSSION THREAD

**Discussion by ingeniumstudio** [ImageMagick/ImageMagick#6129] [Help]
### ImageMagick version

7.1.0.62

### Operating system, version and so on

Android, Termux

### Description

I am trying to crop the top and bottom of images like this to keep only the contents. Image:

![out](https://user-images.githubusercontent.com/117122269/222874464-612a22cc-2315-4593-bc37-eec1119818f6.png)

I would like to cut the header and the footer before trimming. I'm using:

```
convert -density 288 "ElementaryAlgebra2e-WEB_EjIP4sI.pdf[167]" -trim -resize 50% out.png
```

How can I do this?

Thanks

**Comment by snibgo** [comment]
You could "`-crop`" and "`+repage`" the image before the "`-trim`". See https://imagemagick.org/script/command-line-options.php#crop
